src/commands/deployment/__tests__/__snapshots__/gate.test.ts.snap

```diff
@@ -1,14 +1,36 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`gate execute evaluation errors on gate evaluation request should fail when gate evaluation request fails with 500 and fail-on-error is true 1`] = `
+exports[`gate execute evaluation errors on gate evaluation request should fail when gate evaluation request fails with 400 1`] = `
```

Comment:
This is a new snapshot (I missed this in my previous PR). The updated snapshot is just below